### ISSUE TITLE
USWDS - Bug: Improve touch gestures in date picker

### ISSUE THREAD

**Issue by amyleadem** [uswds/uswds#5503]
## Summary
This issue originates from [this comment](https://github.com/uswds/uswds/pull/5374#pullrequestreview-1615663080) in https://github.com/uswds/uswds/pull/5374:

The date picker component cannot be navigated easily with touch navigation with VoiceOver Gestures in iOS Safari. 

Here are some of the items that were discovered: 
- :warning: The initial readout provides keyboard instructions instead of touch instructions
- :warning: In iOS with VoiceOver gestures, the readout when navigating through dates using left and right swipe now includes the abbreviations for the day of the week before the date “Th - sept 28”, “Fr - sept 29”. It is not immediately clear what these letters mean (”Thursday” and “Friday”) and it would be better if they were not read out. (This issue does not happen in `develop`)
- :warning: Up and down swipe in iOS with VO Gestures do not work. Reads out “Heading not found” and does not move focus.
-  :warning: I could not discover an easy way to navigate into the top navigation bar (could only swipe left/right).
- 💡 It would be great if the text input for date picker opened the number keypad. Right now it opens the standard alpha keypad.

**Comment by amycole501** [comment]
@alex-hull we should confirm this is still an issue after we released an updated date picker last year.

**Comment by alex-hull** [comment]
Will confirm this for this next sprint. If any needed steps need to be taken, I will move this to the appropriate party.

**Comment by dct2023** [comment]
@alex-hull Have we confirmed if this is an issue still? If so, is this something we need for upcoming deadlines? Should this be moved to the next sprint since it is currently slated for this sprint.

**Comment by alex-hull** [comment]
@dct2023 We moved these into Amy's sprint board for confirmation on her end. So they were not on my radar this sprint. @amycole501 I can take a look at the bug issues we moved into your queue tomorrow morning.

**Comment by dct2023** [comment]
@alex-hull Thank you! Following up to see if we need anything else for this one. I see it's still in Needs Refinement status so wanted to update if you're working on this one.

**Comment by alex-hull** [comment]
@dct2023 I have not gotten to this one yet, but will confirm what's going on with this today!

**Comment by alex-hull** [comment]
Can confirm that a few of the issues addressed are present. Notes below:

1. Yes, the initial instructions readout for the calendar are for keyboard. Wondering if this could be disorienting to a VO user on mobile - touch instructions are stated when in the text input
2. Yes, the abbreviations for days are read out to the screen reader. It seems efficient in theory, but sounds like a random letter in the middle the middle of the callout
3. I could not replicate the heading not being read out. The header rows of the calendar are read out as such. The month and year are not read as a header for me.
4. Swiping up and down read out the individual letters of February for me, even though I was in the middle of the calendar. I'm unsure if this is typical of VO, or truly a bug. I would think since a keyboard user can use their up and down arrow keys, a VO user could do the same thing.

@dct2023 How should I label this? Testing for mobile and alternative assistive tech will begin in this next sprint. Is it worth pushing this issue (and the date picker a11y test issue) into this upcoming sprint?
